iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ShuntCompensatorImpl.java

```diff
@@ -102,19 +125,87 @@ public double getCurrentB() {
         return bPerSection * getCurrentSectionCount();
     }
 
+    @Override
+    public TerminalExt getRegulatingTerminal() {
+        return regulatingTerminal;
+    }
+
+    @Override
+    public ShuntCompensatorImpl setRegulatingTerminal(Terminal regulatingTerminal) {
+        ValidationUtil.checkRegulatingTerminal(this, (TerminalExt) regulatingTerminal, getNetwork());
+        Terminal oldValue = this.regulatingTerminal;
+        this.regulatingTerminal = regulatingTerminal != null ? (TerminalExt) regulatingTerminal : getTerminal();
+        notifyUpdate("regulatingTerminal", oldValue, this.regulatingTerminal);
+        return this;
+    }
+
+    @Override
+    public boolean isVoltageRegulatorOn() {
+        return voltageRegulatorOn.get(network.get().getVariantIndex());
+    }
+
+    @Override
+    public ShuntCompensatorImpl setVoltageRegulatorOn(boolean voltageRegulatorOn) {
+        int variantIndex = network.get().getVariantIndex();
+        ValidationUtil.checkVoltageControl(this, voltageRegulatorOn, targetV.get(variantIndex));
+        boolean oldValue = this.voltageRegulatorOn.set(variantIndex, voltageRegulatorOn);
+        String variantId = network.get().getVariantManager().getVariantId(variantIndex);
+        notifyUpdate("regulating", variantId, oldValue, voltageRegulatorOn);
```

Comment:
Fix the name of this property: in GeneratorImpl, we use `voltageRegulatorOn`. If we consider regulating is better, change the name of the property everywhere.

To be honest, I think the property name should be the same than the name of the method / attribute. What do you think?

Reply:
Yes, I think I've forgotten to change this in the last commit.